pkg/cmd/cli/cmd/version.go

```diff
@@ -126,13 +126,14 @@ func (o VersionOptions) RunVersion() error {
 		}
 
 		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
-		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err)) {
+		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err)) {
```

Comment:
No worries, here's the `--loglevel=10` output:

`$ oc login -u developer`
`$ oc version --loglevel=10`
```
oc v1.3.0-alpha.0+f2ac159-dirty
kubernetes v1.3.0+57fb9ac
features: Basic-Auth
I0728 18:07:32.674349   13549 loader.go:330] Config loaded from file /home/jvallejo/Documents/go/src/github.com/openshift/origin/openshift.local.config/master/admin.kubeconfig
I0728 18:07:32.674974   13549 round_trippers.go:299] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: oc/v1.3.0+57fb9ac (linux/amd64) kubernetes/57fb9ac" -H "Authorization: Bearer 8u242_OKlT-6gNAZ5oMWaInl3yZ1-wGvxdodDHZKWeU" https://10.13.137.149:8443/api
I0728 18:07:32.698810   13549 round_trippers.go:318] GET https://10.13.137.149:8443/api 200 OK in 23 milliseconds
I0728 18:07:32.698836   13549 round_trippers.go:324] Response Headers:
I0728 18:07:32.698843   13549 round_trippers.go:327]     Cache-Control: no-store
I0728 18:07:32.698849   13549 round_trippers.go:327]     Content-Type: application/json
I0728 18:07:32.698855   13549 round_trippers.go:327]     Date: Thu, 28 Jul 2016 22:07:32 GMT
I0728 18:07:32.698861   13549 round_trippers.go:327]     Content-Length: 136
I0728 18:07:32.698908   13549 request.go:901] Response Body: {"kind":"APIVersions","versions":["v1"],"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]}
I0728 18:07:32.699103   13549 round_trippers.go:299] curl -k -v -XGET  -H "User-Agent: oc/v1.3.0+57fb9ac (linux/amd64) kubernetes/57fb9ac" -H "Accept: application/json, */*" -H "Authorization: Bearer 8u242_OKlT-6gNAZ5oMWaInl3yZ1-wGvxdodDHZKWeU" https://10.13.137.149:8443/apis
I0728 18:07:32.700191   13549 round_trippers.go:318] GET https://10.13.137.149:8443/apis 200 OK in 1 milliseconds
I0728 18:07:32.700204   13549 round_trippers.go:324] Response Headers:
I0728 18:07:32.700212   13549 round_trippers.go:327]     Content-Type: application/json
I0728 18:07:32.700218   13549 round_trippers.go:327]     Date: Thu, 28 Jul 2016 22:07:32 GMT
I0728 18:07:32.700222   13549 round_trippers.go:327]     Content-Length: 1382
I0728 18:07:32.700225   13549 round_trippers.go:327]     Cache-Control: no-store
I0728 18:07:32.700260   13549 request.go:901] Response Body: {"kind":"APIGroupList","groups":[{"name":"apps","versions":[{"groupVersion":"apps/v1alpha1","version":"v1alpha1"}],"preferredVersion":{"groupVersion":"apps/v1alpha1","version":"v1alpha1"},"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]},{"name":"authentication.k8s.io","versions":[{"groupVersion":"authentication.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"authentication.k8s.io/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]},{"name":"autoscaling","versions":[{"groupVersion":"autoscaling/v1","version":"v1"}],"preferredVersion":{"groupVersion":"autoscaling/v1","version":"v1"},"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]},{"name":"batch","versions":[{"groupVersion":"batch/v1","version":"v1"},{"groupVersion":"batch/v2alpha1","version":"v2alpha1"}],"preferredVersion":{"groupVersion":"batch/v1","version":"v1"},"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]},{"name":"extensions","versions":[{"groupVersion":"extensions/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"extensions/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"10.13.137.149:8443"}]}]}
I0728 18:07:32.700960   13549 round_trippers.go:299] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: oc/v1.3.0 (linux/amd64) openshift/f2ac159" -H "Authorization: Bearer 8u242_OKlT-6gNAZ5oMWaInl3yZ1-wGvxdodDHZKWeU" https://10.13.137.149:8443/oapi
I0728 18:07:32.701905   13549 round_trippers.go:318] GET https://10.13.137.149:8443/oapi 200 OK in 0 milliseconds
I0728 18:07:32.701918   13549 round_trippers.go:324] Response Headers:
I0728 18:07:32.701922   13549 round_trippers.go:327]     Cache-Control: no-store
I0728 18:07:32.701927   13549 round_trippers.go:327]     Content-Type: application/json
I0728 18:07:32.701930   13549 round_trippers.go:327]     Date: Thu, 28 Jul 2016 22:07:32 GMT
I0728 18:07:32.701934   13549 round_trippers.go:327]     Content-Length: 93
I0728 18:07:32.701962   13549 request.go:901] Response Body: {"kind":"APIVersions","apiVersion":"v1","versions":["v1"],"serverAddressByClientCIDRs":null}
I0728 18:07:32.702122   13549 round_trippers.go:299] curl -k -v -XGET  -H "User-Agent: oc/v1.3.0 (linux/amd64) openshift/f2ac159" -H "Accept: application/json, */*" -H "Authorization: Bearer 8u242_OKlT-6gNAZ5oMWaInl3yZ1-wGvxdodDHZKWeU" https://10.13.137.149:8443/version/openshift
I0728 18:07:32.702927   13549 round_trippers.go:318] GET https://10.13.137.149:8443/version/openshift 200 OK in 0 milliseconds
I0728 18:07:32.702941   13549 round_trippers.go:324] Response Headers:
I0728 18:07:32.702947   13549 round_trippers.go:327]     Cache-Control: no-store
I0728 18:07:32.702955   13549 round_trippers.go:327]     Content-Type: application/json
I0728 18:07:32.702960   13549 round_trippers.go:327]     Date: Thu, 28 Jul 2016 22:07:32 GMT
I0728 18:07:32.702965   13549 round_trippers.go:327]     Content-Length: 142
I0728 18:07:32.702991   13549 request.go:901] Response Body: {
  "major": "1",
  "minor": "3+",
  "gitCommit": "efda5a8",
  "gitVersion": "v1.3.0-alpha.0+efda5a8",
  "buildDate": "2016-07-28T17:55:12Z"
}
I0728 18:07:32.703120   13549 round_trippers.go:299] curl -k -v -XGET  -H "User-Agent: oc/v1.3.0+57fb9ac (linux/amd64) kubernetes/57fb9ac" -H "Authorization: Bearer 8u242_OKlT-6gNAZ5oMWaInl3yZ1-wGvxdodDHZKWeU" -H "Accept: application/json, */*" https://10.13.137.149:8443/version
I0728 18:07:32.703931   13549 round_trippers.go:318] GET https://10.13.137.149:8443/version 200 OK in 0 milliseconds
I0728 18:07:32.703946   13549 round_trippers.go:324] Response Headers:
I0728 18:07:32.703952   13549 round_trippers.go:327]     Date: Thu, 28 Jul 2016 22:07:32 GMT
I0728 18:07:32.703957   13549 round_trippers.go:327]     Content-Length: 235
I0728 18:07:32.703961   13549 round_trippers.go:327]     Cache-Control: no-store
I0728 18:07:32.703965   13549 round_trippers.go:327]     Content-Type: application/json
I0728 18:07:32.703993   13549 request.go:901] Response Body: {
  "major": "1",
  "minor": "3",
  "gitVersion": "v1.3.0+57fb9ac",
  "gitCommit": "57fb9ac",
  "gitTreeState": "clean",
  "buildDate": "2016-07-28T17:55:12Z",
  "goVersion": "go1.6.3",
  "compiler": "gc",
  "platform": "linux/amd64"
}

Server https://10.13.137.149:8443
OpenShift v1.3.0-alpha.0+efda5a8
Kubernetes v1.3.0+57fb9ac
```